collect_app/src/test/java/org/odk/collect/android/logic/FormControllerTest.java

```diff
@@ -0,0 +1,231 @@
+package org.odk.collect.android.logic;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.FormIndex;
+import org.javarosa.core.model.instance.TreeReference;
+import org.javarosa.form.api.FormEntryCaption;
+import org.javarosa.form.api.FormEntryController;
+import org.javarosa.form.api.FormEntryModel;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class FormControllerTest {
+
+    @Mock
+    File mockMediaFolder;
+    @Mock
+    File mockInstancePath;
+    @Mock
+    FormEntryController mockFormEntryController;
+    @Mock
+    FormEntryModel mockFormEntryModel;
+    @Mock
+    FormIndex mockFormIndex;
+    @Mock
+    FormDef mockForm;
+
+
+
+    // UUT
+    private FormController formController;
```

Comment:
Adding `@InjectMocks` here will allow the Mockito Runner to create this automatically for you using the previously defined Mocks.